newsletters/views.py

```diff
@@ -40,7 +46,9 @@ class Update(PermissionRequiredMixin, generic.edit.UpdateView):
 	raise_exception = True
 
 	def form_valid(self, form):
-		if form.instance.ispublished: form.instance.pub_date = timezone.now()
+		if form.instance.ispublished and not Newsletter.objects.get(id=form.instance.id).ispublished:
```

Comment:
Might be worth adding a comment that this will update whenever the newsletter goes from unpublished to published?

Reply:
(that's what it's doing, right? :P)

Reply:
Correct, and done :man_health_worker: